### DISCUSSION TITLE
Testing if you can get galaxy brain with your own discussion

### DISCUSSION THREAD

**Discussion by arbso** [Schweinepriester/github-profile-achievements#7] [Q&A] [answered]
Testing if you can get galaxy brain with your own discussion

**Comment by arbso** [comment, marked as answer]
Test
edit: Appearantly I can't mark this as an answer

**Reply by owl4ce** [reply]
Change the category to QnA.

**Reply by arbso** [reply]
That did the trick. Not sure if I didn't get the achievement at all or it just takes a while to update. Thanks!

**Reply by augustobr1to** [reply]
it works?

**Reply by RatulHasan** [reply]
That is cool to know! Thank you! :)

**Reply by Coolbreeze151** [reply]
Curious to know if your answer in the comments of a selected answer would be considered as answered as well?

**Comment by payrolldevsys** [comment]
My knowledge is that it doesn't work. Did it work for you?

**Reply by augustobr1to** [reply]
nop

**Comment by qiqt** [comment]
Should work, because this answer shows in your profile. Probably making one more discussion would make you get a badge

**Reply by augustobr1to** [reply]
I dont think so

**Comment by jazzybruno** [comment]
I dont think so

**Comment by jazzybruno** [comment]
ok

**Comment by augustobr1to** [comment]
works

**Comment by MuratAlpTR** [comment]
works

**Comment by lincolnvc** [comment]
that works

**Comment by brunomart97** [comment]
Yes. That works.

**Comment by Suburbanno** [comment]
lel

**Comment by emanmunir** [comment]
it doesnt work?
